[T7][W13-C1]

jessidew95 opened the pull request.

okkhoy commented on docs/AboutUs.md:
```diff
@@ -4,49 +4,35 @@ We are a team based in the [School of Computing, National University of Singapor
+#### [Desmond Ang](https://github.com/KnewYouWereTrouble)
+<img src="images/Desmond.jpg" width="150"><br>
+**Role**: Team Lead, Git Expert <br>
+**Responsibilities**: Deliverables and Deadlines
```
in the context of this project, responsibility means, who is in charge of which component?
who is doing UI, who is doing logic etc.

okkhoy commented on docs/UserGuide.md:
```diff
@@ -10,125 +10,190 @@
+
+* **`search`**` tomorrow`<br>
+**`tag`**` 2 Homework`<br>
+Deletes the task with `ID #2` in the DoDo-Bird.
```
ouch! you ask for tagging a task and the app deletes the task??!! :-1:

okkhoy commented on docs/UserGuide.md:
```diff
@@ -10,125 +10,190 @@
+Examples:
+
+* **`add`**` Meet with professor; CS1234;`
+* **`add`**` Complete tutorial activites; CS1234; from 25/10/17 23:59; till 18/10/17 12:00;`
```
btw., do you allow adding tasks that have overlapping times?

KnewYouWereTrouble replied:
Will implement defensibility mechanism to prevent this !

okkhoy reviewed (commented):
hey guys,
you have a fair attempt at the user guide. 
- you don't seem to have a generic list command that provides the user to view all tasks at once.
- there are numerous writing errors, please proof read the document thoroughly before submitting.
- there is some bit of duplication in the description and the quoted text throughout the document, retain which is most apt and remove the others.

your UI mock up seems to have eliminated the browser panel, was it a conscious decision? its OK to remove the browser, you can use that space to give some meaningful information to the user in that area in the GUI.

cheers,
Akshay

okkhoy commented:
@jessidew95 @chuayupeng @KnewYouWereTrouble @langyanbin0314 
Some comments added. Please ack & close the PR after reading comments.

KnewYouWereTrouble commented on docs/UserGuide.md:
```diff
@@ -10,125 +10,190 @@
+Format:
+
+* `add TASKNAME; [a line of details;] [from DD/MM/YY [hh:mm];] [till DD/MM/YY [hh:mm];]` <br>
+* `add TASKNAME; [a line of details;] [DD/MM/YY;]`
```
Good point ! Will consider it as an enhancement !

KnewYouWereTrouble commented:
Acknowledged ! Thanks for the feedback !

theapeeeee committed:
Add Recurrence class to model recurrences

theapeeeee committed:
Able to add recurrence to task

theapeeeee committed:
Implement Undo Redo Commands

theapeeeee committed:
Implement default time and date

theapeeeee committed:
Allow specifying of by time if on dates are specified

theapeeeee committed:
Change modelling of recurrence

theapeeeee committed:
Implement Adding, Updating, Deleting of Recurrence

chuayupeng committed:
Added Summary Panel, changed up UI

chuayupeng committed:
Added Summary Panel, changed up UI

theapeeeee committed:
Fix NullPointerException at SummaryPanel.java

theapeeeee committed:
Fix NullPointerException at SummaryPanel.java

chuayupeng committed:
Minor changes

chuayupeng committed:
Minor changes

theapeeeee committed:
Connect TodayList in model to UI

theapeeeee committed:
Connect TodayList in model to UI

theapeeeee committed:
Remove some unused code

theapeeeee committed:
Fix code style

chuayupeng committed:
Summary Panel updated to show any tasks that has today in either on or by dates

chuayupeng committed:
Summary Panel updated to show any tasks that has today in either on or by dates

chuayupeng committed:
Fixed <!-- error

chuayupeng committed:
Fixed <!-- error

theapeeeee committed:
Fix UI Tests

chuayupeng committed:
fixed css extentions and theme paths

chuayupeng committed:
fixed css extentions and theme paths

Commit:
Merge pull request #68 from CS2103AUG2016-W13-C1/Fix-UI-Tests

Fix UI Tests

Commit:
Merge pull request #65 from CS2103AUG2016-W13-C1/Undo-Redo-Commands

Implement Undo Redo Commands

chuayupeng committed:
Merge pull request #66 from CS2103AUG2016-W13-C1/Recurring-Tasks

Recurring tasks

theapeeeee committed:
Fix failing XmlToDoListStorageTest

chuayupeng committed:
Merge branch 'Summary-Panel' of https://github.com/CS2103AUG2016-W13-C1/main into Summary-Panel

# Conflicts:
#	src/main/resources/view/MainWindow.fxml

chuayupeng committed:
Minor Edits on refactoring test case

langyanbin0314 committed:
added priority function

langyanbin0314 committed:
changed task card

langyanbin0314 committed:
modified taskcard

theapeeeee committed:
Fix Update Command for updating priority
Fix TestTasks to accomodate priority

theapeeeee committed:
Allow add priority on floating tasks

theapeeeee committed:
Implement adding priority for floating tasks

theapeeeee committed:
Merge branch 'Summary-Panel'

Conflicts:
	src/main/java/seedu/todo/model/Model.java
	src/main/java/seedu/todo/model/ModelManager.java
	src/main/java/seedu/todo/ui/MainWindow.java

Commit:
Merge pull request #69 from CS2103AUG2016-W13-C1/Summary-Panel

Summary panel

langyanbin0314 committed:
added new regex for search priority

langyanbin0314 committed:
modified search priority

langyanbin0314 committed:
added color scheme for different priority

langyanbin0314 committed:
modified variable

langyanbin0314 committed:
improved codacy

langyanbin0314 committed:
improve codacy

langyanbin0314 committed:
improved codacy

langyanbin0314 committed:
improved codacy

langyanbin0314 committed:
improve codacy

langyanbin0314 committed:
adjusted priority circle position

langyanbin0314 committed:
fixed the problem where user can only update tasks with single digit index

theapeeeee committed:
Fix missing update for priority

theapeeeee committed:
Fix coding style

langyanbin0314 committed:
added a test for default Priority

theapeeeee committed:
Merge branch 'priority-tasks' of https://github.com/CS2103AUG2016-W13-C1/main into priority-tasks

theapeeeee committed:
Merge remote-tracking branch 'origin/priority-tasks'

Conflicts:
	.project
	src/main/java/seedu/todo/model/Model.java
	src/main/java/seedu/todo/model/ModelManager.java

theapeeeee committed:
Fix unable to add floating tasks with details

theapeeeee committed:
Add collate comments and update documentation

langyanbin0314 committed:
update command summary

langyanbin0314 committed:
Merge branch 'master' of https://github.com/CS2103AUG2016-W13-C1/main

* 'master' of https://github.com/CS2103AUG2016-W13-C1/main:
  Add collate comments and update documentation
  Fix unable to add floating tasks with details

chuayupeng committed:
Fixed On Date after current date but still shows in summary panel bug

langyanbin0314 committed:
modified user guide

chuayupeng committed:
Merge branch 'Summary-Panel'

langyanbin0314 committed:
modified command summary

chuayupeng committed:
Merge branch 'master' of https://github.com/CS2103AUG2016-W13-C1/main

langyanbin0314 committed:
modified command summary

langyanbin0314 committed:
Merge branch 'master' of https://github.com/CS2103AUG2016-W13-C1/main

* 'master' of https://github.com/CS2103AUG2016-W13-C1/main:
  Fixed On Date after current date but still shows in summary panel bug

langyanbin0314 committed:
changed command summary

chuayupeng committed:
Minor edits to test cases

chuayupeng committed:
Merge branch 'master' of https://github.com/CS2103AUG2016-W13-C1/main

langyanbin0314 committed:
added GUI

langyanbin0314 committed:
Merge branch 'master' of https://github.com/CS2103AUG2016-W13-C1/main

* 'master' of https://github.com/CS2103AUG2016-W13-C1/main:
  Minor edits to test cases

langyanbin0314 committed:
added screenshot

langyanbin0314 committed:
added  screenshots

langyanbin0314 committed:
added screenshot

theapeeeee committed:
Fix .project

theapeeeee committed:
Update README.md

chuayupeng committed:
Fix to be able to see Start and End for task properly

chuayupeng committed:
Updated Summary Card to reflect priority level

jessidew95 committed:
create TagCard and TagListPanel

langyanbin0314 committed:
collate

jessidew95 committed:
create getUnmodifiableTagList function

jessidew95 committed:
add TagListPanel to MainWindow

jessidew95 committed:
add getName function in Tags

Commit:
Merge pull request #72 from CS2103AUG2016-W13-C1/create-tag-list

Create tag list

theapeeeee committed:
Add Collate tags

theapeeeee committed:
Fix authoring

langyanbin0314 committed:
collate fxml

jessidew95 committed:
Collate

jessidew95 committed:
Merge branch 'master' of https://github.com/CS2103AUG2016-W13-C1/main

# Conflicts:
#	src/main/resources/view/TaskListCard.fxml

theapeeeee committed:
Perform Collate and Update AboutMe.md

langyanbin0314 committed:
added screen shots

langyanbin0314 committed:
Merge branch 'master' of https://github.com/CS2103AUG2016-W13-C1/main

* 'master' of https://github.com/CS2103AUG2016-W13-C1/main:
  Collate

langyanbin0314 committed:
Merge branch 'master' of https://github.com/CS2103AUG2016-W13-C1/main

* 'master' of https://github.com/CS2103AUG2016-W13-C1/main:
  Perform Collate and Update AboutMe.md

# Conflicts:
#	docs/UserGuide.md

langyanbin0314 committed:
solve merge conflict

langyanbin0314 committed:
Update UserGuide.md

langyanbin0314 committed:
modified user guide

theapeeeee committed:
Update documentaion
Credit Natty Date Parser in README.md
Fix links in AboutMe.md that links to collated files

theapeeeee committed:
Update collated code links in AboutUs.md

theapeeeee committed:
Refactor Model and ModelManager
Apply separation of concerns lower the coupling between ModelManager and
internal DoDoBird
Limit exposure of internal DoDoBird as much as possible

Rename ToDoList to DoDoBird as a clearer name
Refactor DoDoBird by applying SLAP

Enable undo for tags

Fix Connection between TagList in UI to internal tag list in DoDoBird

theapeeeee committed:
Fix coding style and update comments for various commands

theapeeeee committed:
Implement on date cannot be after by date

theapeeeee committed:
Implement add task to index 0 of list

jessidew95 committed:
Shift tagListPanel to left of main window

jessidew95 committed:
Fix UI layout

jessidew95 committed:
Fix anchorman alignment

theapeeeee committed:
Add test cases for Undo, Reset

chuayupeng committed:
Added Next 7 Days feature

theapeeeee committed:
Update and create new test cases

theapeeeee committed:
Fix LogicManagerTest when testing for adding recurring tasks

okkhoy commented on collated/main/A0093896H.md:
```diff
@@ -0,0 +1,2010 @@
+    public final int targetIndex;
+    public final UniqueTagList tags;
+    
+    public TagCommand(int targetIndex, String tagNames) throws Exception {
```
the exception is too generic here; can consider adding something more specific to indicate the kind of exception

okkhoy commented on collated/main/A0093896H.md:
```diff
@@ -0,0 +1,2010 @@
+        this.targetIndex = targetIndex;
+        
+        if (tagNames.isEmpty()) {
+            throw new Exception();
```
especially since, nothing is passed to the exception!

okkhoy commented on collated/main/A0093896H.md:
```diff
@@ -0,0 +1,2010 @@
+            for (Tag tag : tags) {
+                try {
+                    toTag.addTag(tag);
+                } catch (DuplicateTagException e) {}
```
catch and do what?

okkhoy commented on collated/main/A0093896H.md:
```diff
@@ -0,0 +1,2010 @@
+    public final int targetIndex;
+    public final UniqueTagList tags;
+    
+    public UntagCommand(int targetIndex, String tagNames) throws Exception {
```
same as previous

okkhoy commented on collated/main/A0093896H.md:
```diff
@@ -0,0 +1,2010 @@
+import seedu.todo.model.task.UniqueTaskList.TaskNotFoundException;
+
+/**
+ * Update a task identified using it's last displayed index from the to do list.
```
Update_**s**_ ...

okkhoy commented on collated/main/A0093896H.md:
````diff
@@ -0,0 +1,2010 @@
+``` java
+    public ModelManager(ReadOnlyToDoList initialData, UserPrefs userPrefs) {
+        toDoList = new ToDoList(initialData);
+        filteredTasks = new FilteredList<>(toDoList.getTasks());
````
can include the type information for the list here

okkhoy commented on collated/main/A0093896H.md:
```diff
@@ -0,0 +1,2010 @@
+
+import seedu.todo.model.task.ReadOnlyTask;
+
+public class CompletedQualifier implements Qualifier{
```
header comments?

okkhoy commented on collated/main/A0093896H.md:
```diff
@@ -0,0 +1,2010 @@
+import seedu.todo.model.task.ReadOnlyTask;
+
+public class CompletedQualifier implements Qualifier{
+    private boolean wantsDone;
```
consider renaming the variable to make it more meaningful at first look!

okkhoy commented on collated/main/A0093896H.md:
```diff
@@ -0,0 +1,2010 @@
+
+public class Completion {
+
+    private boolean completed;
```
please follow the suggested naming convention for boolean variables

okkhoy commented on collated/main/A0093896H.md:
```diff
@@ -0,0 +1,2010 @@
+            }
+            break;
+        default :
+            //NONE - do nothing
```
should probably be some exception.

okkhoy commented on collated/test/A0093896H.md:
```diff
@@ -0,0 +1,523 @@
+public class AddCommandTest extends ToDoListGuiTest {
+    
+    @Test
+    public void add() {
```
please follow the guidelines for test naming

okkhoy commented on collated/test/A0093896H.md:
````diff
@@ -0,0 +1,523 @@
+```
+###### \java\seedu\todo\commons\util\DateTimeUtilTest.java
+``` java
+package seedu.todo.commons.util;
````
shouldn't this be in a `test` package?

okkhoy commented on collated/test/A0093896H.md:
```diff
@@ -0,0 +1,523 @@
+    public void execute_unmark_successful() throws Exception {
+        // setup expectations
+        TestDataHelper helper = new TestDataHelper();
+        Task toBeMarked = helper.generateFullTask(0);
```
consider moving these to the test setup

okkhoy commented on collated/test/A0093896H.md:
```diff
@@ -0,0 +1,523 @@
+    @Test
+    public void execute_search_matchesIfDone() throws Exception {
+        TestDataHelper helper = new TestDataHelper();
+        Task t1 = helper.generateTaskWithName("bla bla KEY bla");
```
these are definitely duplicated code; use `@Before` and `@After` as necessary and move them to a common setup.

okkhoy commented on collated/main/A0121643R.md:
````diff
@@ -0,0 +1,722 @@
+```
+###### \java\seedu\todo\logic\parser\ToDoListParser.java
+``` java
+    private Command prepareAdd(String args) {
````
try improving the SLAP on this method

okkhoy commented on collated/main/A0121643R.md:
```diff
@@ -0,0 +1,722 @@
+                ParserFormats.ADD_TASK_ARGS_RECUR_FORMAT_BY, ParserFormats.ADD_TASK_ARGS_RECUR_FORMAT_ON, 
+                ParserFormats.ADD_TASK_ARGS_FORMAT_FT, ParserFormats.ADD_TASK_ARGS_FORMAT_BY, 
+                ParserFormats.ADD_TASK_ARGS_FORMAT_ON, ParserFormats.ADD_TASK_ARGS_FORMAT_FLOAT};
+        //add buy chicken on friday by wednesday priority high every year ; chicken kfc
```
errr? remove unrelated comments from the code

okkhoy commented on collated/main/A0121643R.md:
```diff
@@ -0,0 +1,722 @@
+                }
+            }
+        }
+        if (tempArgs.indexOf("tag") == 0) {
```
magic numbers all over the place here! please fix that

okkhoy commented on collated/main/A0121643R.md:
````diff
@@ -0,0 +1,722 @@
+```
+###### \java\seedu\todo\model\task\Priority.java
+``` java
+public class Priority {
````
header comments?

okkhoy commented on collated/main/A0121643R.md:
````diff
@@ -0,0 +1,722 @@
+###### \java\seedu\todo\model\task\Task.java
+``` java
+    /**
+     * Copy constructor.
````
not a very useful comment

okkhoy commented on collated/main/A0121643R.md:
```diff
@@ -0,0 +1,722 @@
+    }
+    
+    /**
+     * Add a tag to the task's tag list 
```
Add_**s**_ ...
similarly for other comments.

okkhoy commented on collated/main/A0138967J.md:
```diff
@@ -0,0 +1,371 @@
+import seedu.todo.commons.util.DateTimeUtil;
+import seedu.todo.model.task.ReadOnlyTask;
+
+public class TodayDateQualifier implements Qualifier{
```
header comments?

okkhoy commented on collated/main/A0138967J.md:
```diff
@@ -0,0 +1,371 @@
+            super.updateItem(task, empty);
+            boolean dateCheck = false;
+            try{
+                dateCheck = LocalDate.now().isAfter(task.getByDate().getDate()) || LocalDate.now().isBefore(task.getOnDate().getDate());
```
this is a very long line, please split it appropriately

okkhoy commented on collated/main/A0138967J.md:
```diff
@@ -0,0 +1,371 @@
+            if (empty || task == null) {
+                setGraphic(null);
+                setText(null);
+            } else if(!dateCheck){
```
need a space between `){`

okkhoy commented on collated/main/A0142421X.md:
```diff
@@ -0,0 +1,265 @@
+        this.placeHolderPane = pane;
+    }
+
+    public static TagListPanel load(Stage primaryStage, AnchorPane tagListPanelPlaceholder,
```
try a better split?

okkhoy commented on collated/main/A0142421X.md:
```diff
@@ -0,0 +1,265 @@
+
+    public static TagListPanel load(Stage primaryStage, AnchorPane tagListPanelPlaceholder,
+    								ObservableList<Tag> tagList){
+    	TagListPanel tagListPanel =
```
spacing seems off here

okkhoy commented on collated/main/A0142421X.md:
```diff
@@ -0,0 +1,265 @@
+    private void setConnections(ObservableList<Tag> tagList) {
+        tagListView.setItems(tagList);
+        tagListView.setCellFactory(listView -> new TagListViewCell());
+        //setEventHandlerForSelectionChangeEvent();
```
remove commented out code; you can retrieve it from rcs later if necessary

okkhoy commented on collated/main/A0142421X.md:
```diff
@@ -0,0 +1,265 @@
+        });
+    }
+    
+    class TagListViewCell extends ListCell<Tag> {
```
header comments?

Commit:
Merge pull request #74 from CS2103AUG2016-W13-C1/Extend-Testing

Extend testing

Commit:
Merge pull request #73 from CS2103AUG2016-W13-C1/add-panel

Shift tagListPanel to left of main window

jessidew95 committed:
Fixed indentation and removed commented out code

jessidew95 committed:
Add comments

jessidew95 committed:
refactor person/people to task

jessidew95 committed:
refactor person to task in comments

jessidew95 committed:
refactor person to task

jessidew95 committed:
Create Exit Handle

jessidew95 committed:
Create Tag Card Handle

jessidew95 committed:
Create Tag List Panel Handle

jessidew95 committed:
Add Tag List Panel Handle to Main Gui Handle

theapeeeee committed:
Fix coding style and improve exception handling

theapeeeee committed:
Add tasks count to tags

theapeeeee committed:
Prettify date printing and add Ui for recurrence

chuayupeng committed:
Fixed Next 7 Days to exclude today's tasks

jessidew95 committed:
Change UI layout

jessidew95 committed:
add tag field

jessidew95 committed:
Change text when task is completed

jessidew95 committed:
change UI in readme

jessidew95 committed:
change width of tag card

jessidew95 committed:
update UI in readme

chuayupeng committed:
Changed Logo

Commit:
Merge pull request #81 from CS2103AUG2016-W13-C1/add-num-task-tag-panel

Add num task tag panel

chuayupeng committed:
Updated Logo

theapeeeee committed:
Implement searching for done, undone, tasks

chuayupeng committed:
update gitignore to include bin images

chuayupeng committed:
Merge branch 'master' of https://github.com/CS2103AUG2016-W13-C1/main

theapeeeee committed:
Fix regex

chuayupeng committed:
Edit to fix PrettifyDate

theapeeeee committed:
Fix unmark test in LogicManagerTest.java

Commit:
Merge pull request #82 from CS2103AUG2016-W13-C1/improve-search

Improve search

theapeeeee committed:
Fix update command on date cannot be before end date

jessidew95 committed:
Edit UI layout

jessidew95 committed:
Edit GUI tests

langyanbin0314 committed:
changed path

langyanbin0314 committed:
refactored, removed unnecessary import

langyanbin0314 committed:
removed unnecessary imports

chuayupeng committed:
Merge branch 'master' of https://github.com/CS2103AUG2016-W13-C1/main

langyanbin0314 committed:
fixed add and update bug

Commit:
Merge pull request #86 from CS2103AUG2016-W13-C1/update-date

fixed add and update bug

langyanbin0314 committed:
added new test for priorityqulifier and datetimeutilqualifier

theapeeeee committed:
Fix code style and refactor LogicManagerTest.java

theapeeeee committed:
Merge branch 'master' into more-test

Conflicts:
	src/main/java/seedu/todo/commons/util/DateTimeUtil.java

theapeeeee committed:
Merge branch 'more-test'

theapeeeee committed:
Merge branch 'master' into change-ui-layout

Conflicts:
	src/main/java/seedu/todo/ui/TaskCard.java
	src/main/resources/view/TagCard.fxml
	src/main/resources/view/TaskListCard.fxml
	src/test/java/guitests/guihandles/TaskCardHandle.java

theapeeeee committed:
Fix UI issues

theapeeeee committed:
Merge branch 'change-ui-layout'

langyanbin0314 committed:
removed priority circle for completed task

langyanbin0314 committed:
deleted unnecessary import, added collate

jessidew95 committed:
Fix alignment

theapeeeee committed:
Extend Testing

Commit:
Merge pull request #88 from CS2103AUG2016-W13-C1/fix-tag-and-command-box

Fix alignment in tag card and shift command box down

theapeeeee committed:
Fix Tag indexing

langyanbin0314 committed:
syc today and this week list for mark and unmark

langyanbin0314 committed:
modified untag

chuayupeng committed:
Changed Help URL to DoDo Bird User Guide

chuayupeng committed:
Added TodayQualifierTest, fixed minor bug in Today Selection

theapeeeee committed:
Implement event handling for saveLocationChangedEvent

theapeeeee committed:
Implement event handling for saveLocationChangedEvent in Reset Command

theapeeeee committed:
Refactor initialized method in TaskCard.java

theapeeeee committed:
Implement mark command to update recurring dates

theapeeeee committed:
Remove See command, make search default to search for uncompleted task

theapeeeee committed:
Implement selection event for tag panel

theapeeeee committed:
Improve user feedback message

chuayupeng committed:
Updated coding violations

theapeeeee committed:
Improve task card Date UI

theapeeeee committed:
Improve search feedback message to users

theapeeeee committed:
Fix random colors for tag list

theapeeeee committed:
Improve Search

theapeeeee committed:
Update documentation

theapeeeee committed:
Refector Add command

theapeeeee committed:
Collate codes

theapeeeee committed:
Update documentation

theapeeeee committed:
Fix UserGuide image path

theapeeeee committed:
Fix UserGuide image size

theapeeeee committed:
Improve test coverage

langyanbin0314 committed:
changed ui for overdue tasks

theapeeeee committed:
Fix code style and include Manual Testing script

theapeeeee committed:
Fix MarkLogicTest

Commit:
Merge pull request #90 from CS2103AUG2016-W13-C1/overdue-tasks

changed ui for overdue tasks

theapeeeee committed:
Fix code style

langyanbin0314 committed:
added unused code

langyanbin0314 committed:
Merge branch 'master' of https://github.com/CS2103AUG2016-W13-C1/main

* 'master' of https://github.com/CS2103AUG2016-W13-C1/main:
  Fix code style

# Conflicts:
#	src/main/java/seedu/todo/ui/TaskCard.java

langyanbin0314 committed:
added author and about us

chuayupeng committed:
Up coverage for mark command

theapeeeee committed:
Add interactivity to today and next 7 days panel

theapeeeee committed:
Add author tags for Collate

chuayupeng committed:
Added author to markTodayTask fix

theapeeeee committed:
Merge branch 'master' of https://github.com/CS2103AUG2016-W13-C1/main

theapeeeee committed:
update documentation with new images

theapeeeee committed:
Fix dev guide

langyanbin0314 committed:
shift the priority circle

theapeeeee committed:
Fix MarkLogicTest

theapeeeee committed:
Add sampletest data

theapeeeee committed:
Improve test coverage

theapeeeee committed:
Improve test coverage

theapeeeee committed:
Fix DateTimeUtilTest

theapeeeee committed:
Improve test coverage

theapeeeee committed:
Fix RecurrenceTest

jessidew95 committed:
Cover StringUtil tests

jessidew95 committed:
fix unreachable code

jessidew95 committed:
add dateTimeUtilTests

jessidew95 committed:
remove extra lines

jessidew95 committed:
collate unused

jessidew95 committed:
Correct typo

jessidew95 committed:
add author

jessidew95 committed:
return original code

jessidew95 committed:
label as unused

theapeeeee committed:
Implement after updating commands, still retain current filtered list

jessidew95 committed:
Comment out test case and author as unused

theapeeeee committed:
Improve test coverage

theapeeeee committed:
Add CardStylerTest

Commit:
Merge pull request #95 from CS2103AUG2016-W13-C1/authorDewi

add author

Commit:
Merge pull request #94 from CS2103AUG2016-W13-C1/collateUnused

Added some unused handles

Commit:
Merge pull request #93 from CS2103AUG2016-W13-C1/testcases

TestCases

theapeeeee committed:
Improve test coverage

Commit:
Merge pull request #96 from CS2103AUG2016-W13-C1/refresh-filteredlist

Implement after updating commands, still retain current filtered list

jessidew95 committed:
Edit Documentation

jessidew95 committed:
Edit User Guide

jessidew95 committed:
Edit User Guide

jessidew95 committed:
Fix formatting of user guide

jessidew95 committed:
Edit user guide

jessidew95 committed:
Add images to User Guide

jessidew95 committed:
Edit User Guide format

langyanbin0314 committed:
add collate

langyanbin0314 committed:
Merge branch 'master' of https://github.com/CS2103AUG2016-W13-C1/main

* 'master' of https://github.com/CS2103AUG2016-W13-C1/main:
  Edit User Guide format
  Add images to User Guide
  Edit user guide
  Fix formatting of user guide
  Edit User Guide
  Edit User Guide
  Edit Documentation

langyanbin0314 committed:
add collate for docs

chuayupeng committed:
add WeekQualifierTest

langyanbin0314 committed:
Merge branch 'master' of https://github.com/CS2103AUG2016-W13-C1/main

* 'master' of https://github.com/CS2103AUG2016-W13-C1/main:
  add WeekQualifierTest

langyanbin0314 committed:
collate files

jessidew95 committed:
Edit User Guide

jessidew95 committed:
add line break in user guide

langyanbin0314 committed:
modify test script

langyanbin0314 committed:
Merge branch 'master' of https://github.com/CS2103AUG2016-W13-C1/main

* 'master' of https://github.com/CS2103AUG2016-W13-C1/main:
  add line break in user guide
  Edit User Guide

chuayupeng committed:
Fixed Typo in dev guide

chuayupeng committed:
Dev Guide Typo Fix

chuayupeng committed:
Dev Guide Typo Fix

chuayupeng committed:
Typo Fix

theapeeeee committed:
Fix testscript instructions

theapeeeee committed:
Update testscript for help reset and store commands

theapeeeee committed:
Fix testscript

theapeeeee committed:
Fix testscript newline
